front-end/package.json

```diff
@@ -17,10 +17,10 @@
     "styled-components": "^5.1.0"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "cross-env NODE_ENV=development PORT=3001 react-scripts start",
```

Comment:
Can you check in your Windows environment if you really do need to set NODE_ENV? I thought react-scripts would handle that?

Reply:
Yeah I'll double check

Reply:
Yeah you're right, worked fine without it. Updated